File: miracle-extra/src/main/java/com/miracle/extra/service/ErpInfoService.java
package com.miracle.extra.service;

import com.miracle.extra.domain.ErpInfo;

import java.util.List;

public interface ErpInfoService {

    public List<ErpInfo> selectWorkerList(ErpInfo erpInfo);

    public List<ErpInfo> selectWorkshopList(ErpInfo erpInfo);

    public List<ErpInfo> selectWorkshopByCode(String workshopCode);

    public List<ErpInfo> selectWorkshopByName(String workshopName);

    public List<ErpInfo> selectWorkerByCode(String workerCode);

    public List<ErpInfo> selectWorkerByName(String workerName);

    public List<ErpInfo> searchWorkshop(String searchText);

    public List<ErpInfo> searchWorker(String searchText);

    public List<ErpInfo> searchMaterial(String searchText);
}

File: miracle-extra/src/main/java/com/miracle/extra/service/ErpInfoService.kt
package com.miracle.extra.service;

import com.miracle.extra.domain.ErpInfo


interface ErpInfoService {

    fun selectWorkerList(erpInfo: ErpInfo): List<ErpInfo>;

    fun selectWorkshopList(erpInfo: ErpInfo): List<ErpInfo>;

    fun selectWorkshopByCode(workshopCode: String?): List<ErpInfo>;

    fun selectWorkshopByName(workshopName: String?): List<ErpInfo>;

    fun selectWorkerByCode(workerCode: String?): List<ErpInfo>;

    fun selectWorkerByName(workerName: String?): List<ErpInfo>;

    fun searchWorkshop(searchText: String?): List<ErpInfo>;

    fun searchWorker(searchText: String?): List<ErpInfo>;

    fun searchMaterial(searchText: String?): List<ErpInfo>;
}
